container/container.go

```diff
@@ -109,6 +109,7 @@ type Container struct {
 
 	// Fields here are specific to Windows
 	NetworkSharedContainerID string
+	SharedEndpointList       []string
```

Comment:
It doesn't seem like either of these fields need to be persisted on the container object. Can make sure the fields aren't marshaled to json?